ProgramFlow/ProgramFlow/Flow.cs

```diff
@@ -18,30 +12,58 @@ public static class Flow
         /// <returns><b>true</b> if <paramref name="value"/> is <b>false</b>; <b>false</b> if <paramref name="value"/> is <b>true</b>.</returns>
         public static bool InvertBoolean(bool value)
         {
-            // TODO: Fix this method so it passes all associated tests.
-            return value;
+            //Return the inverted boolean.
+            return !value;
         }
 
         /// <summary>
         /// Returns the correct <see cref="DriverAction"/> value for each <see cref="LightColor"/> using an if-else statement.
         /// </summary>
-        /// <param name="value">The <see cref="LightColor"/> for which to return a <see cref="DriverAction"/>.</param>
+        /// <param name="color">The <see cref="LightColor"/> for which to return a <see cref="DriverAction"/>.</param>
         /// <returns>The correct <see cref="DriverAction"/> value.</returns>
         public static DriverAction DriveSafelyIfElse(LightColor color)
         {
-            // TODO: Fix this method so it passes all associated tests. An if-else statement must be used to return the correct values.
-            return (DriverAction)24;
+            var driverAction = DriverAction.Unknown;
+
+            if (color == LightColor.Green)
+            {
+                driverAction = DriverAction.ProceedWithCaution;
+            }
+            else if (color == LightColor.Yellow)
+            {
+                driverAction = DriverAction.StopIfSafe;
+            }
+            else if (color == LightColor.Red)
+            {
+                driverAction = DriverAction.Stop;
+            }
+
+            return driverAction;
         }
 
         /// <summary>
         /// Returns the correct <see cref="DriverAction"/> value for each <see cref="LightColor"/> using a switch statement.
         /// </summary>
-        /// <param name="value">The <see cref="LightColor"/> for which to return a <see cref="DriverAction"/>.</param>
+        /// <param name="color">The <see cref="LightColor"/> for which to return a <see cref="DriverAction"/>.</param>
         /// <returns>The correct <see cref="DriverAction"/> value.</returns>
         public static DriverAction DriveSafelySwitch(LightColor color)
         {
-            // TODO: Fix this method so it passes all associated tests. A switch statement must be used to return the correct values.
-            return (DriverAction)24;
+            var driverAction = DriverAction.Unknown;
+
+            switch (color)
+            {
+                case LightColor.Green:
+                    driverAction = DriverAction.ProceedWithCaution;
```

Comment:
Same here: You could (if you want) just do:

return DriverAction.ProceedWtihCaution;

I'm fine either way, so choose whichever you want going forward.